e2e/swapSheetFlow.spec.js

```diff
@@ -421,6 +443,27 @@ describe('Swap Sheet Interaction Flow', () => {
   it('Should show Swap Settings State on Settings Button press', async () => {
     await Helpers.waitAndTap('exchange-settings-button');
     await Helpers.checkIfVisible('swap-settings-header');
+    await Helpers.waitAndTap('swap-settings-routes-label');
+    await Helpers.checkIfVisible('explain-sheet-routeSwaps');
+    await Helpers.swipe('explain-sheet-routeSwaps', 'down', 'slow');
+    await Helpers.waitAndTap('swap-settings-routes-current-rainbow');
+    await Helpers.tapByText('0x');
+    await Helpers.checkIfVisible('swap-settings-routes-current-0x');
+    await Helpers.waitAndTap('swap-settings-flashbots-switch-false');
+    await Helpers.checkIfVisible('swap-settings-flashbots-switch-true');
+    await Helpers.waitAndTap('swap-settings-flashbots-label');
+    await Helpers.checkIfVisible('explain-sheet-flashbots');
+    await Helpers.swipe('explain-sheet-flashbots', 'down', 'slow');
+    await Helpers.clearField('swap-slippage-input-2');
+    await Helpers.typeText('swap-slippage-input-', '10', false);
+    await Helpers.checkIfVisible('swap-slippage-input-10');
+    await Helpers.waitAndTap('swap-settings-defaults-button');
+    await Helpers.checkIfVisible('swap-settings-routes-current-rainbow');
+    await Helpers.checkIfVisible('swap-settings-flashbots-switch-false');
+    await Helpers.checkIfVisible('swap-slippage-input-2');
+    await Helpers.waitAndTap('swap-slippage-label');
+    await Helpers.checkIfVisible('explain-sheet-slippage');
+    await Helpers.swipe('explain-sheet-slippage', 'down', 'slow');
```

Comment:
looking nice, could you update the name of this test to say this is happening on mainnet? it wasn't clear to me until i saw it running